### DISCUSSION TITLE
Sector active status NO

### DISCUSSION THREAD

**Discussion by btclinux** [filecoin-project/lotus#9223] [Storage Provider] [answered]
After sealing I see OnChain "YES" status but sectors still inactive after 1 hour. Am I missing something? What I can find in logs? Or just wait?

ID  State    OnChain  Active  Expiration                    Deals  
0   Proving  YES      NO      3659288 (in 1 year 24 weeks)  CC     
1   Proving  YES      NO      3659225 (in 1 year 24 weeks)  CC     
2   Proving  YES      NO      3659350 (in 1 year 24 weeks)  CC

**Comment by TippyFlitsUK** [comment, marked as answer]
You need to wait for the new sectors to complete a `WindowPoST` before they are active on-chain @btclinux .

**Reply by btclinux** [reply]
How to watch this process?

**Reply by TippyFlitsUK** [reply]
`lotus-miner proving deadlines` will show you which deadline your sectors belong to and which deadline is currently queued for `WindowPoST`.
